collections/_posts/2024-01-30-brain-imaging-generation.md

```diff
@@ -0,0 +1,149 @@
+    title: "Brain Imaging Generation with Latent Diffusion Models"
+    venue: "Arxiv 2022"
+pdf: "https://arxiv.org/pdf/2209.07162.pdf"
+---
+
+# Notes
+
+* The generated synthetic dataset is available at [Academics Torrents](https://academictorrents.com/details/63aeb864bbe2115ded0aa0d7d36334c026f0660b) 
+* An inspired code is available on the following [Monai link](https://monai.io/model-zoo.html)
+
+&nbsp;
+
+# Highlights
+
+* Application of the Latent Diffusion Model framework for MR image synthesis of the human brain
+* An encoder/decoder model dedicated to brain MRI reconstruction is proposed
+* Inverstigation of the application of conditioning to age, gender, ventricular volumes and brain volumes 
+
+* The model achieves new SOTA for brain MR image synthesis
+* A synthetic dataset of 100,000 volumes together with the conditioning information are publicly available
+
+&nbsp;
+
+# Introduction
+
+* The objective of the paper is to generate realistic large scale dataset with additional related "low dimensional" information such as age, sex or volumes.
+
+* 31,740 T1w 3D MR images from the UK Biobank datas are used during training 
+
+* One interest of such dataset would to provide enough data to learn to retrieve the age of a patient based on its brain MR image.
+
+&nbsp;
+
+# Methodology
+
+## Preprocessing steps
+
+* An existing network called [UnitRes](https://github.com/brudfors/UniRes) was used to perform a rigid body registration to a common MNI space
+
+* The final images are resampled to a uniform resolution of $$1 \, mm^3$$ 
+
+* The images are all cropped to a consistent volume size of $$160 \times 224 \times 160$$ voxels
+
+&nbsp;
+
+## LDM  architecture
+
+* The method is directly inspired by the [latent diffusion model](https://creatis-myriad.github.io/2023/12/19/latent-diffusion-models.html) whose architecture is summarized below:   
+
+![](/collections/images/latent-DM/latent-DM-architecture.jpg)
+
+&nbsp;
+
+* The autoencoder is first trained with a combination of L1 loss, perceptual loss, a patch-based adversarial objective and a KL regularization of the latent space
+
+* The encoder maps the brain image to a latent representation with a size of $$20 \times 28 \times 20$$ voxels
+
+* The diffusion model is then trained using $$1000$$ steps for the Markov chain process 
+
+* The model is conditioned according to age, gender, ventricular volume and brain volume
+
+* The conditioning is performed bu combining the concatenation of the conditioning with the input data and the use of cross-attention mechanisms
+
+![](/collections/images/latent-DM/cross-attention.jpg)
+
+&nbsp;
+
+
+# Results
+
+* The autoencoder compressed each dimension of the input data by a factor of 8
+* DDIM is used during inference to reduce from $$1000$$ to $$50$$ the number of time steps during sampling
```

Comment:
I would add that this  reduce the average sampling time from $$142 \pm 1.6$$s to $$7.6 \pm 0.2$$s.

Reply:
Great ;)